packages/dds/tree/src/treeFactory.ts

```diff
@@ -8,27 +8,170 @@ import type {
 	IChannelFactory,
 	IFluidDataStoreRuntime,
 	IChannelServices,
+	IChannelStorageService,
 } from "@fluidframework/datastore-definitions/internal";
+import type {
+	ITelemetryContext,
+	IExperimentalIncrementalSummaryContext,
+	ISummaryTreeWithStats,
+	IRuntimeMessageCollection,
+} from "@fluidframework/runtime-definitions/internal";
+
+import type { ISequencedDocumentMessage } from "@fluidframework/driver-definitions/internal";
 import type { SharedObjectKind } from "@fluidframework/shared-object-base";
 import {
+	type IFluidSerializer,
+	type ISharedObject,
 	type ISharedObjectKind,
+	SharedObject,
 	createSharedObjectKind,
 } from "@fluidframework/shared-object-base/internal";
 
-import {
-	SharedTree as SharedTreeImpl,
-	type ISharedTree,
-	type SharedTreeOptions,
-	type SharedTreeOptionsInternal,
+import type {
+	SchematizingSimpleTreeView,
+	SharedTreeContentSnapshot,
+	SharedTreeOptions,
+	SharedTreeOptionsInternal,
+	SharedTreeEditBuilder,
+	SharedTreeChange,
+	ITreePrivate,
 } from "./shared-tree/index.js";
-import type { ITree } from "./simple-tree/index.js";
-
+import type {
+	ImplicitFieldSchema,
+	ITree,
+	ReadSchema,
+	SimpleTreeSchema,
+	TreeView,
+	TreeViewConfiguration,
+	UnsafeUnknownSchema,
+	VerboseTree,
+} from "./simple-tree/index.js";
 import { SharedTreeFactoryType, SharedTreeAttributes } from "./sharedTreeAttributes.js";
+import { Breakable } from "./util/index.js";
+import { SharedTreeKernel } from "./shared-tree/index.js";
+import { UsageError } from "@fluidframework/telemetry-utils/internal";
+import { fail } from "@fluidframework/core-utils/internal";
+import type { SharedTreeCore } from "./shared-tree-core/index.js";
+
+/**
+ * {@link ITreePrivate} extended with ISharedObject.
+ * @remarks
+ * This is used when integration testing this package with the Fluid runtime as it exposes the APIs the runtime consumes to manipulate the tree.
+ */
+export interface ISharedTree extends ISharedObject, ITreePrivate {}
+
+/**
+ * Shared object wrapping {@link SharedTreeKernel}.
+ */
+class SharedTreeImpl extends SharedObject implements ISharedTree {
```

Comment:
Moved unchanged, except for new name (to not collide with SharedTree export in this file) and no longer file exported.